app/main/views/briefs.py

```diff
@@ -352,7 +352,7 @@ def list_brief_services(brief_id):
     brief = Brief.query.filter(
         Brief.id == brief_id
     ).filter(
-        Brief.status == "live"
+        Brief.status != "draft"
```

Comment:
For this API endpoint I think yes - the restriction is only there to avoid KeyErrors on incomplete drafts.  It's up to the frontend apps to decide whether to show users something relating to a withdrawn brief, but I don't think there's any reason to stop the API from making the info about it available to the frontend apps.